turbinia/e2e/googleclouddisk.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# This Turbinia e2e test will test the googleclouddisk functionality.
+# It will try to analyse the disk 'test-disk2' in zone 'us-central1-a'.
+
+
+TURBINIA_CLI="turbiniactl"
+MAIN_LOG="main.log"
+STATUS_LOG="status.log"
+STATS_LOG="stats.log"
+DETAIL_LOG="jobdetails.log"
+OUT_TGZ="jobresults.tgz"
+DISK="test-disk2"
+ZONE="us-central1-a"
+
+echo "Creating GCE test disk to use in e2e test"
+gcloud compute disks create $DISK --image=$DISK --zone=$ZONE
+
+echo "Executing googlecloudisk e2e test....this takes ~60 minutes!"
+$TURBINIA_CLI -L $MAIN_LOG -a -w googleclouddisk -d $DISK -z $ZONE &
```

Comment:
Why run this in the background? If you just let it run in the foreground then you wouldn't need the loop down below.

Reply:
As I wanted to use the status command to get some more information during the e2e test I needed the request id which is in the output of the main command. I can not parse output of the main command (to get the request_id) and run multiple status commands if the main command is running in the foreground. As I can now use the --request_id to set my own request id I can rewrite this to run in the foreground.